SparkPost/2web2ui
# FE-1098 - Incorporate UX feedback for the events feature

[FE-1098](https://jira.int.messagesystems.com/browse/FE-1098)

### What Changed
- Made updates to the events list and details pages, incorporating UX feedback listed in the ticket

### How To Test
- Sign in with the usual account!
- Go to `/reports/message-events` and verify the following changes were made:
  - [x] The "Add Filters" and "Share" buttons use the `"secondary"` variant
  - [x] The "Remove" and "Add Filter" icons within the "Add Filters" modal were updated
  - [x] The `<Select/>` components used in the "Add Filters" modal are wider, allowing content to be read
  - [x] The form filtering `<Panel/>` and the `<TableCollection/>` are now connected
- Go to `/reports/message-events/details/<eventID>` and verify the following changes were made:
  - [x] There is a section divider below each `<Panel/>` title (note, the way I did it was hacky - I opened a feature request for Matchbox to make this better: https://github.com/SparkPost/matchbox/issues/467)
  - [x] When an event is selected in the "Message History" table, the font weight is semibold and the color is `blue.700`
  - [x] Verify the labels paired with form fields in the "Event Details" `<Panel/>` are vertically centered
 
### To Do
